web-console/package.json

```diff
@@ -28,44 +29,47 @@
     "hjson": "^3.1.2",
     "lodash.debounce": "^4.0.8",
     "numeral": "^2.0.6",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.3",
+    "react-ace": "^6.4.0",
+    "react-addons-css-transition-group": "^15.6.2",
+    "react-dom": "^16.8.3",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "react-table": "^6.8.6",
+    "react-table": "^6.9.2",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
     "@types/classnames": "^2.2.7",
-    "@types/d3-array": "^1.2.4",
-    "@types/jest": "^23.3.13",
-    "@types/lodash.debounce": "^4.0.4",
-    "@types/mocha": "^5.2.5",
-    "@types/node": "^10.12.18",
+    "@types/d3-array": "^1.2.6",
+    "@types/jest": "^24.0.6",
+    "@types/lodash.debounce": "^4.0.5",
+    "@types/mocha": "^5.2.6",
```

Comment:
Are all of these version updates necessary, and can we use fixed package versions instead of "^"?

I think it would be simpler for review and also tracking our dependencies/licenses if we minimize version upgrades

Reply:
I just upgraded to the latest versions as part of my testing. I figured since I was messing with the package versions anyway I would update things to the latest versions as I was going to be doing comprehensive end-to-end testing anyway. As for using the non-fixed versions that is not an issue as there is a package-lock file included which locks those versions.

Reply:
also almost all of the 'spurious' things I upgraded are devDependancies in the normal dependency list every change is needed for this PR except `@types/hjson` which should actually probably be in the dev list also.

Reply:
Moved `@types/hjson` to dev list

Reply:
removed all the package changes other than what is necessary for this PR

Reply:
Cool, thanks